curriculum/challenges/english/blocks/zh-a1-learn-certification-introduction/691fd70b91fd33630effdf6e.md

```diff
@@ -0,0 +1,110 @@
+---
+id: 691fd70b91fd33630effdf6e
+title: Pinyin-Tone Input
+challengeType: 22
+dashedName: pinyin-tone-input
+lang: zh-CN
+inputType: pinyin-tone
+---
+
+<!-- (audio) Wang Hua: 你好 (nǐ hǎo)，世界 (shì jiè)。-->
+
+# --description--
+
+Chinese is spoken in many regional varieties. In this curriculum, you will learn **Mandarin**, the most widely used standard form of spoken Chinese.
+
+Chinese is written in two major forms: Simplified and Traditional Chinese. This curriculum will focus on **Simplified Chinese**.
+
+Since Chinese characters are not phonetic, a system called **Pinyin**​ is used to represent their sounds. Pinyin is a romanization system that uses the Latin alphabet to indicate how Chinese characters are pronounced. It serves as an essential tool for beginners to learn the pronunciation of the language and to type Chinese on digital devices.
+
+In the audio, Wang Hua is saying "Hello, world." in Chinese. You're about to input your first two Chinese words, each made of two characters. Sounds unbelievable, right? freeCodeCamp has customized the **Pinyin-Tone Input** and **Pinyin-Hanzi Input** to help you do this easily.
+
+Try typing `ni3 hao3` and `shi4 jie4` to see what happens. If you make a mistake, just delete the incorrect part and type again.
+
+Remember to expand the explanation section below to see what's happening behind the scenes.
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
```

Comment:
I'm using a PC. At the moment, I have to copy and paste `nǐ hǎo` and `shì jiè` in the answers for it to work, as typing `ni3 hao3` and `shi4 jie4` did not produce any changes. I'm wondering whether the "How to Set Up the Chinese Keyboard on Your Computer" and "How to Set Up the Chinese Keyboard on Mobile" should come prior to starting filling in the blanks. If they do, they are out of order in the structure file.